packages/web-app-files/tests/unit/views/spaces/Project.spec.js

```diff
@@ -231,6 +231,8 @@ function getMountedWrapper(spaceResources = [], spaceItem = null, imageContent =
       $client: {
         files: {
           getFileContents: jest.fn().mockImplementation(() => Promise.resolve('filecontent')),
+          // TODO: Add & reference space fixtures in line below
```

Comment:
@JanAckermann there's some test warnings about not being able to access the `fileInfo` in the view. I think it'd make a lot of sense to add spaces fixtures and import them here